### PR TITLE
Doc update versions

### PR DESCRIPTION
**Pull request by b-gent**
This is release-specific so the change is going straight to release branch.

@carlescufi  @rlubos please confirm that these are the correct versions for the release:
Zephyr - 2.0.99
mcuboot - 1.4.0
NCS - 1.1.0
nrfxlib - 1.1.0

### CODE REVIEW

File: doc/nrf/index.rst [hunk 1]
```diff
@@ -13,6 +13,7 @@ The |NCS| includes the Zephyr™ real-time operating system (RTOS), which is bui
 Documentation for different versions of the |NCS| is available at the following links:
 
 * `nRF Connect SDK latest documentation`_ (from the master branch)
+* `nRF Connect SDK v1.1.0 documentation`_
```
Comment (ru-fu) [inline doc/nrf/index.rst]: Is that link defined already?

**Comment by ru-fu** [comment]
There are mentions of v1.0.0 in the installation instructions (gs_ins_windows); these should be updated.
Also in ug_dev_model, but that one is more generic, so I think it's okay to leave it at v1.0.0.

**Comment by ru-fu** [comment]
Nordic Thingy:52 link is dead.
firmware.nrfcloud.com link as well - this isn't used anymore and can be deleted.

File: doc/nrf/index.rst (hunk 1 above)
Comment (b-gent) [reply doc/nrf/index.rst]: forgot to git add, fixed now

**Comment by b-gent** [comment]
> Nordic Thingy:52 link is dead.
> firmware.nrfcloud.com link as well - this isn't used anymore and can be deleted.

The moved it from Development Kits to Prototyping Platforms on the website. Fixed.

**Comment by b-gent** [comment]
> There are mentions of v1.0.0 in the installation instructions (gs_ins_windows); these should be updated.
> Also in ug_dev_model, but that one is more generic, so I think it's okay to leave it at v1.0.0.

done

**Comment by carlescufi** [comment]
> This is release-specific so the change is going straight to release branch.
> 
> @carlescufi @rlubos please confirm that these are the correct versions for the release:
> Zephyr - 2.0.99
> mcuboot - 1.4.0
> NCS - 1.1.0
> nrfxlib - 1.1.0

MCUboot is actually 1.4.99 as per the tag. We should've really tagged it 1.4.0 (@rlubos  noticed this), but we made a mistake, might as well use the same version everywhere.
The rest of the versions are correct AFAICT.

**Comment by b-gent** [comment]
> MCUboot is actually 1.4.99 as per the tag.

MCUboot version changed to 1.4.99

**Pull request merged**